server/controllers/stock/reports/stock/lots_report.js

```diff
@@ -60,14 +61,16 @@ function stockLotsReport(req, res, next) {
     'enterprisePurchaseInterval', 'exhausted', 'expired',
     'inventory_uuid', 'lifetime_lot', 'min_delay',
     'min_months_security_stock', 'mvt_quantity', 'near_expiration',
-    'purchase_interval', 'tag_name', 'tracking_consumption',
-    'tracking_expiration', 'wac'];
+    'purchase_interval', 'tag_name', 'tracking_consumption', 'wac'];
 
   const dateKeys = ['min_stock_date', 'max_stock_date'];
 
   return Stock.getLotsDepot(null, options)
     .then((rows) => {
       rows.forEach(row => {
+        const current = new Date();
+        const delay = moment(new Date(row.expiration_date)).diff(current);
+        row.delay_expiration = moment.duration(delay).humanize(true);
```

Comment:
If this is going to be printed, I would be careful about using `humanize()`.  We don't want an administrator to see an old report and think "this expires in two months" when the stock is already expired.  Always include the expiration date itself as well, for context.

Also, are we sure `humanize()` will respect the user's choice of language?

Reply:
Ok I see I agree with you in the idea of keeping the context, but` humanize()` works very well with the user's choice of language